app.py

```diff
@@ -0,0 +1,110 @@
+    if request.method == "POST":
+        # Not using .get method to keep dataType consistent
+        id = ""
+        if "id" in data:
+            id = data["id"]
+        
+        # Specifies a functionality for the request
+        operation = ""
+        if "operation" in data:
+            operation = data["operation"]
+
+        # Parses the pages innerHTML and finds the questions and answer input elements
+        # Uses LLM prompting
+        if operation == "Question-Answer-LLM":
+            # This functionality is the following two operations combined
+            htmlContent = data["requestData"]
+            prompts = question_answer_prompts(id, htmlContent)
+            responses = question_answer_prompting(id, prompts)
+            return json.dumps(qas)
+        
+        if operation == "Question-Answer-Prompts":
+            # "Question-Answer-LLM" but only returns the prompts
+            htmlContent = data["requestData"]
+            prompts = question_answer_prompts(id, htmlContent)
+            return json.dumps(prompts)
+        
+        if operation == "Question-Answer-Prompting":
+            # "Question-Answer-LLM" but only returns the prompts
+            prompts = data["requestData"]
+            responses = question_answer_prompting(id, prompts)
+            return json.dumps(responses)
+        
+        # Parses the pages innerHTML and finds the questions and answer input elements
+        # Uses same parent element to find question answer pairs, must faster
+        if operation == "Question-Answer-Fast":
+            htmlContent = data["requestData"]
+            qas = question_answer_fast(id, htmlContent)
+            return json.dumps(qas)
+        
+        if operation == "Question-Answer-Fast-2":
+            # Gets the question tag and superset question tags
+            # Name, NameYour Answer are both returned
+            htmlContent = data["requestData"]
+            qas = question_answer_fast(id, htmlContent,False)
+            return json.dumps(qas)
+        
+        # Answers questions given and id for an account with stored information
+        if operation == "Answer-Input-Questions":
+            qas = data["requestData"]
+            answers = answer_input_questions(id, qas)
+            return json.dumps(answers)
+        
+        if operation == "Add-User-QAs":
+            # Not implemented
+            return json.dumps({})
+    
+    return json.dumps([None])
+
+
+@app.route("/", methods=("GET", "POST"))
+def index():
+    data = request.json
+    if request.method == "POST":
+        id = ""
+        if "id" in data:
+            id = data["id"]
+        # Find the operation requested
+        operation = ""
+        if "operation" in data:
+            operation = data["operation"]
+
+    return "Hello, World!"
+
+
+if __name__ == "__main__":
+    ##
+    path = "c:/Users/bill/github/AutofillAPI"
```

Comment:
a) Use the `os` and relative paths for this kind of stuff
b) Use the `dotenv` package. https://pypi.org/project/python-dotenv/. Very simple to use:
```
from dotenv import load_dotenv

load_dotenv()  # take environment variables from .env.

# Code of your application, which uses environment variables (e.g. from `os.environ` or
# `os.getenv`) as if they came from the actual environment.
```

Reply:
Done!